### PR TITLE
{bio}[foss/2019b] TopHat v2.1.2

### PR DESCRIPTION
**Pull request by hcartiaux**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by hcartiaux** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-023 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/29f4d5176db92e6ea49fb6fa72aa4af7 for a full test report.

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.1.2'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = "TopHat is a fast splice junction mapper for RNA-Seq reads."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```
Comment (Micket) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb:16]: I suspect this software has any BLAS/LAPACK, so we can drop this down to the `gompi` level instead.

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+checksums = [
+    '15016b82255dad085d4ee7d970e50f0e53a280d466335553d47790d8344ff4b1',  # v2.1.2.tar.gz
+    'a33075c2affda995ddbd250933dbcaf94f68768b1eebead58a69705fc40306ef',  # tophat-2.0.13-zlib.patch
+]
+
+builddependencies = [('Autoconf', '2.69')]
+
+dependencies = [
+    ('Boost', '1.71.0', '', ('gompi', '2019b')),
```
Comment (Micket) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb:33]: You shouldn't need to specify this. Even if this was on foss-2019b level, it should automatically pick up dependencies from gompi, GCC, GCCcore levels.
```suggestion
    ('Boost', '1.71.0'),
```

Comment (hcartiaux) [reply easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb]: Thanks for your comment. I've applied your suggested change.

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb:16]: This isn't resolved? Or does it have a dependency on BLAS/LAPACK that i missed?

Comment (smoors) [reply easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb:16]: @hcartiaux what @Micket is suggesting is to change this line to:
```
toolchain = {'name': 'gompi', 'version': '2019b'}
```

**Comment by hcartiaux** [comment]
I implemented the suggested changes.
Sorry for the delays, I'm still learning easybuild ;)

**Comment by lexming** [comment]
Re-triggering travis

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node379.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/500d58c3b5997d12ba96245a5eb40784 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node120.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/de1addeb96bb953a55cd89c2f107e1f8 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @hcartiaux !

**Pull request merged**